admin_interface/templates/admin/change_list.html

```diff
@@ -0,0 +1,25 @@
+{% extends "admin/change_list.html" %}
+{% load i18n admin_list admin_interface_tags %}
+{# copied from django 4.0.7 #}
+
+      {% block filters %}
+        {% if cl.has_filters %}
+          <div id="changelist-filter">
+            <h2>{% translate 'Filter' %}</h2>
```

Comment:
I forgot to add translators comments to note these strings are already translated in the catalogs for django.contrib.admin!

Reply:
Do you want to add them?